src/components/AdminSettings.vue

```diff
@@ -88,9 +89,77 @@
 					@input="onInput(false)">
 			</div>
 			<div>
-				<h3>
-					{{ t('integration_openai', 'Select which features you want to enable') }}
-				</h3>
+				<h2 class="mid-setting-heading">
+					{{ t('integration_openai', 'Usage limits') }}
+				</h2>
+				<div class="line">
+					<!--Time period in days for the token usage-->
+					<label for="openai-api-quota-period">
+						{{ t('integration_openai', 'Quota enforcement time period (days)') }}
+					</label>
+					<input id="openai-api-quota-period"
+						v-model="state.quota_period"
+						type="number"
+						@input="onInput(false)">
+				</div>
+				<div class="line">
+					<!--Loop through all quota types and list an input for them on this line-->
+					<!--Only enforced if the user has not provided an own API key (in the case of OpenAI)-->
+					<label for="openai-api-quotas">
+						{{ t('integration_openai', 'Usage quotas per time period') }}
+					</label>
+					<table class="quota-table">
+						<thead>
+							<tr>
+								<th width="120px">
+									{{ t('integration_openai', 'Quota type') }}
+								</th>
+								<th>{{ t('integration_openai', 'Per-user quota / period') }}</th>
+								<th v-if="quotaInfo !== null">
+									{{ t('integration_openai', 'Current system-wide usage / period') }}
+								</th>
+							</tr>
+						</thead>
+						<tbody>
+							<tr v-for="(quota,index) in state.quotas" :key="quota.type">
+								<td class="text-cell">
+									{{ t('integration_openai', capitalizedWord(quota.type)) }}
+								</td>
+								<td>
+									<input :id="'openai-api-quota-' + quota.type"
+										v-model="quota.value"
+										:title="t('integration_openai', 'A per-user limit for uasge of this API type (0 for unlimited))')"
+										type="number"
+										@input="onInput(false)">
+									<span v-if="quotaInfo !== null" class="text-cell">
+										{{ t('integration_openai', quotaInfo[index].unit) }}
```

Comment:
This doesn't work. To be translatable, the call to t must be with a string literal as a message